api-extractor/src/generators/ApiFileGenerator.ts

```diff
@@ -67,7 +67,7 @@ export default class ApiFileGenerator extends ApiItemVisitor {
   protected visitApiStructuredType(apiStructuredType: ApiStructuredType): void {
     const declarationLine: string = apiStructuredType.getDeclarationLine();
 
-    if (apiStructuredType.documentation.preapproved) {
+    if (apiStructuredType.documentation && apiStructuredType.documentation.preapproved) {
```

Comment:
Interesting that we need this null check for apiStructuredType.documentation.  How does that happen?